contracts/serverless-v2/Jobs.sol

```diff
@@ -130,11 +130,29 @@ contract CommonChainJobs is
     // jobKey => selectedExecutorAddress => hasExecuted
     mapping(uint256 => mapping(address => bool)) public hasExecutedJob;
 
+    bytes32 private constant DOMAIN_SEPARATOR = 
+        keccak256(
+            abi.encode(
+                keccak256("EIP712Domain(string name,string version)"),
+                keccak256("marlin.oyster.Jobs"),
+                keccak256("1")
+            )
+        );
+    
+    bytes32 private constant RELAY_JOB_TYPEHASH = 
+        keccak256("RelayJob(address operator,uint256 jobId,bytes32 codeHash,bytes codeInputs,uint256 deadline,uint256 jobRequestTimestamp,uint8 sequenceId,address jobOwner)");
```

Comment:
For these 3 type hashes, we can actually skip the operator. Because registration has already been done. and msg_sender can always be checked against the stored value of operator.

Also, even if somebody does the front run of these transaction, they won't get benefit, because stakes are update for stored operator only.